src/plugins/ai_assistant_management/selection/public/routes/components/ai_assistant_selection_page.test.tsx

```diff
@@ -0,0 +1,148 @@
+
+jest.mock('../../app_context');
+
+describe('AiAssistantSelectionPage', () => {
+  const setBreadcrumbs = jest.fn();
+  const navigateToApp = jest.fn();
+
+  const generateMockCapabilities = (hasPersission: boolean) =>
+    ({
+      observabilityAIAssistant: { show: hasPersission },
+      securitySolutionAssistant: { 'ai-assistant': hasPersission },
+    } as unknown as CoreStart['application']['capabilities']);
+
+  const testCapabilities = generateMockCapabilities(true);
+
+  const renderComponent = (capabilities: CoreStart['application']['capabilities']) => {
+    (useAppContext as jest.Mock).mockReturnValue({
+      capabilities,
+      setBreadcrumbs,
+      navigateToApp,
+    });
+    return render(<AiAssistantSelectionPage />, {
+      wrapper: I18nProvider,
+    });
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('sets the breadcrumbs on mount', () => {
+    renderComponent(testCapabilities);
+    expect(setBreadcrumbs).toHaveBeenCalledWith([
+      {
+        text: 'AI Assistant',
+      },
+    ]);
+  });
+
+  it('renders the title and description', () => {
+    renderComponent(generateMockCapabilities(true));
+    expect(screen.getByText('AI Assistant')).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        'AI Assistants use generative AI to help your team by explaining errors, suggesting remediation, and helping you request, analyze, and visualize your data.'
+      )
+    ).toBeInTheDocument();
+  });
+
+  describe('Observability AI Assistant Card', () => {
+    describe('when the feature is disabled', () => {
+      it('displays the disabled callout', () => {
+        renderComponent(generateMockCapabilities(false));
+        expect(screen.getByText('This feature is disabled.')).toBeInTheDocument();
+      });
+    });
+
+    describe('when the feature is enabled', () => {
+      it('does not display the disabled callout', () => {
+        renderComponent(testCapabilities);
+        expect(screen.queryByText('This feature is disabled.')).not.toBeInTheDocument();
+      });
+
+      it('renders the manage settings button', () => {
+        renderComponent(testCapabilities);
+        expect(screen.getByTestId('pluginsAiAssistantSelectionPageButton')).toBeInTheDocument();
+      });
+
+      it('navigates to the observability AI Assistant settings on button click', () => {
+        renderComponent(testCapabilities);
+        fireEvent.click(screen.getByTestId('pluginsAiAssistantSelectionPageButton'));
+        expect(navigateToApp).toHaveBeenCalledWith('management', {
+          path: 'kibana/observabilityAiAssistantManagement',
+        });
+      });
+
+      it('renders the documentation links correctly', () => {
+        renderComponent(testCapabilities);
+
+        expect(
+          screen.getByTestId('pluginsAiAssistantSelectionPageDocumentationLink')
+        ).toHaveAttribute(
+          'href',
+          'https://www.elastic.co/guide/en/observability/current/obs-ai-assistant.html'
```

Comment:
I think we discussed this before, but we shouldn't be pointing to the `current` docs, we should be pointing to the the documentation for the version of the user's deployment. It's good that this test makes it explicit though.

Reply:
Thanks so much for the review @dgieselaar , could you please point me out some example of retrieving the deployment version?

Reply:
The version getDocLinksMeta returns `master` , doesn't match with the renamed `main` branch. 

<img width="2559" alt="Screenshot 2024-07-29 at 18 17 18" src="https://github.com/user-attachments/assets/b074ca1b-aaba-43a4-9e13-029a69485111">

Reply:
After the discussion https://elastic.slack.com/archives/C5TQ33ND8/p1722353968519609, I use `getDocLinks` to generate the links here.